src/content/pages/venue.mdx

```diff
@@ -23,13 +23,13 @@ Address: 5. května 1640/65, 140 21 Praha 4, Nusle
 - **Vyšehrad:** A historic castle complex with parks and beautiful views of Prague. A must-visit.
 - **Náplavka from Vyšehrad:** A lively riverside area with pubs and spots to relax.
 - **I.P. Pavlova:** One metro stop away — great for food, with plenty of restaurants and fast-food options.
-- **City center:** Take the metro two stops to **Muzeum** (line C), then exit onto Wenceslas Square. To explore further, transfer to the green line at **Muzeum** and go to **Staroměstská** (Old Town Square) or **Malostranská** (across the river).
+- **City Centre:** Take the metro two stops to **Muzeum** (line C), then exit onto Wenceslas Square. To explore further, transfer to the green line at **Muzeum** and go to **Staroměstská** (Old Town Square) or **Malostranská** (across the river).
 
 ---
 
 ## Sprints Venue – Saturday & Sunday
 ### Where
-The EuroPython sprints, along with the Unconference day, Humble Data, and Django Girls, will take place at Italská Budova Building, Prague University of Economics and Business, Žižkov Campus on Saturday & Sunday **July 21st to July 22nd**.
+The EuroPython sprints, along with the Unconference day, Humble Data, and Django Girls, will take place at Italská Budova Building, Prague University of Economics and Business, Žižkov Campus on Saturday & Sunday **19–20 July**.
```

Comment:
We might want to remove this for now, it is not certain if it will be hosted there.